pkg/cachedir/manager.go

```diff
@@ -0,0 +1,186 @@
+package cachedir
```

Comment:
I don't think this meets the minimum level of genericity to warrant a unique package. This should just be a part of the distribution package.